lxd/instance/drivers/driver_qemu.go

```diff
@@ -2860,7 +2860,7 @@ if [ ! -e "systemd" ] || [ ! -e "lxd-agent" ]; then
     exit 1
 fi
 
-if [ ! -e "/lib/systemd/system" ]; then
```

Comment:
Could you please add a comment to explain what's happening (as in the commit message) as this is not obvious.